db.go

```diff
@@ -131,6 +136,34 @@ func (rr *redisRepo) fetchInstanceEvents(instanceID string) ([]*lifecycleEvent,
 	return events, nil
 }
 
+func (rr *redisRepo) fetchAllInstanceEvents() (map[string][]*lifecycleEvent, error) {
+	conn := rr.cg.Get()
+	defer rr.closeConn(conn)
+
+	instanceEventKeys, err := redis.Strings(conn.Do("KEYS", fmt.Sprintf("%s:instance:*:events", RedisNamespace)))
```

Comment:
I've fought with `SCAN` for an hour and can't make sense of the raw results, and what relationship they possibly have to what is documented.

The results I'm getting are two `[]uint8`, ftr.